tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -212,6 +212,10 @@ def parameters(cls):
                 'sort_key': StringField(
                     optional=True, default='image_id', choices=['image_id', 'image_size'],
                     description='Key by which annotations will be sorted.'
+                ),
+                'has_background_images': BoolField(
```

Comment:
background as the rule associated with background class, it can confuse if you will use this name for empty images, however it does not add any additional background thing, it just prevent filtering.
```suggestion
                'remove_empty_images': BoolField(
```
or
```suggestion
                'filter_empty_images': BoolField(
```
will be more suitable

P.S. It should be reflected in dataset name in definitions too

Reply:
I'm Ok to use whatever name you propose.

But IMO background is just something that is opposed to the foreground, so those can equally be images or classes depending on the context. So, `has_background` is what is really confusing, it does not hint on what it is actually doing. At the same time `empty` is more like blank, containing no foreground or background. But, once again, you're the owner of the component, so I'll use the name you feel appropriate with here.

I'm also confused with the use-case when you really need to filter some images out of the validation set for MS COCO dataset. All images from the validation set should be used to compute the metric.